statsrunner/loop.py

```diff
@@ -4,16 +4,23 @@
 import json
 import sys
 import traceback
-import decimal
-import argparse
 import statsrunner.shared
 import statsrunner.aggregate
 from statsrunner.common import decimal_default
 
+
 def call_stats(this_stats, args):
+    """Create dictionary of enabled stats for this_stats object.
+
+    Args:
+      this_stats (cls): stats_module that specifies calculations for relevant input, processed by internal methods of process_file().
```

Comment:
Presumably `cls` should be `stats.dashboard.CommonSharedElements`, but I could be wrong ??

Reply:
Also `args` should be listed as an... arg!